hamilton/lifecycle/default.py

```diff
@@ -376,7 +376,10 @@ def run_to_execute_node(
         if node_name not in self.cache_vars:
             return node_callable(**node_kwargs)
 
-        node_hash = graph_types.hash_source_code(node_callable, strip=True)
+        source_of_node_callable = node_callable
+        while hasattr(source_of_node_callable, "func"):  # handle partials
```

Comment:
I this here maybe better to use `isinstance(source_of_node_callable, partial)`, since `func` is a too common name?

Reply:
sure good idea